Modules/HR/Resources/views/mail/infra-team.blade.php

```diff
@@ -0,0 +1,18 @@
+<div>
+    <style>
+        .line {
+            line-height: 1px;
+        }
+    </style>
```

Comment:
Let's avoid using br tag. Instead of it, we can add a custom class here something like
```suggestion
    <style>
        .line {
            line-height: 1px;
        },
        .margin-bottom {
            margin-bottom: 40px;
        }
    </style>
```